Chaincase.SSB/_Imports.razor

```diff
@@ -6,3 +6,5 @@
 @using Microsoft.AspNetCore.Components.Web
 @using Microsoft.JSInterop
 @using Chaincase.UI
+@using BlazorBarcodeScanner.ZXing.JS
```

Comment:
Does this import really belong at the top level?

Reply:
That's what the integration guide states

Reply:
There's also no other place to reference this without coupling the UI with a platform specific implementation

Reply:
ZXingBlazorCameraScanner.rarzor?